posthog/api/user.py

```diff
@@ -334,7 +337,7 @@ def redirect_to_site(request):
 @authenticate_secondarily
 def change_password(request):
     """
-    DEPRECATED: This endpoint has been deprecated in favor of /api/v2/user/ 
+    DEPRECATED: This endpoint has been deprecated in favor of /api/v2/user/
```

Comment:
not sure if this should be changed too, we don't have the change_password at all in `/api/users/` atm

Reply:
I think this function and its path `"api/user/change_password"` can maybe even be just deleted, since we don't use it anywhere anymore (and very unlikely for anyone to be changing their password externally this way).

Reply:
:+1 to @Twixes, nuking it